R/agglomerate.R

```diff
@@ -45,26 +45,26 @@
 #'   to \code{getPrevalence} and \code{getPrevalentTaxa} and used in
 #'   \code{agglomeratebyPrevalence}
 #'   \itemize{
-#'        \item \code{remove_empty_ranks}: A single boolean value for selecting
+#'        \item \code{empty.ranks.rm}: A single boolean value for selecting
 #'        whether to remove those columns of rowData that include only NAs after
-#'        agglomeration. (By default: \code{remove_empty_ranks = FALSE})
-#'        \item \code{make_unique}: A single boolean value for selecting
-#'        whether to make rownames unique. (By default: \code{make_unique = TRUE})
+#'        agglomeration. (By default: \code{empty.ranks.rm = FALSE})
+#'        \item \code{make.unique}: A single boolean value for selecting
+#'        whether to make rownames unique. (By default: \code{make.unique = TRUE})
 #'        \item \code{detection}: Detection threshold for absence/presence.
 #'        Either an absolute value compared directly to the values of \code{x}
-#'        or a relative value between 0 and 1, if \code{as_relative = FALSE}.
+#'        or a relative value between 0 and 1, if \code{as.relative = FALSE}.
```

Comment:
this argument will be removed in a separate PR I think?

Reply:
Yes, https://github.com/microbiome/mia/issues/577